pkg/k8s/run.go

```diff
@@ -18,6 +18,17 @@ import (
 
 // Run runs scan on kubernetes cluster
 func Run(cliCtx *cli.Context) error {
+	// Full-cluster scanning with '--format table' without explicit '--report all' is not allowed so that it won't mess up user's terminal.
+	if cliCtx.String("report") == "all" &&
```

Comment:
make const for `all` and `table` strings

Reply:
sure, but I'll handle it later